en/creating-artboard-sketch.md

```diff
@@ -22,7 +22,7 @@ If the [Titillium Web Font][1] isn't installed on your computer, please do it be
 
 ## Step by step
 
-### 1. Add the **Indigo.Design System** in Sketch.
+### 1. Add the **Indigo.Design System** as a Library.
 
 1.  Download the [**Indigo.Design System**](https://cloud.indigo.design/home).
```

Comment:
On p.2 _Open up the Sketch app on your Mac, click on the Sketch item from the Apple Menu, and select Preferences._ it should be Sketch item NEXT TO the Apple Menu